pkgs/top-level/all-packages.nix

```diff
@@ -15945,6 +15947,15 @@ in
 
   blas = callPackage ../development/libraries/science/math/blas { };
 
+  clblas-cuda = callPackage ../development/libraries/science/math/clblas/cuda {
+    cudatoolkit = pkgs.cudatoolkit75;
+    inherit (linuxPackages) nvidia_x11;
+    python = pkgs.python34Packages.python;
```

Comment:
Using a specific version of a package that is not in `python-packages.nix` is fine.

So `clblas-cuda` is a non-Python library, which needs Python only during build-time, is that correct? If that's the case, then it is preferred if it takes the default Python version and you could drop the line
```python = pkgs.python34Packages.python;```
and possibly also the override of `boost`, because it should already be build with Python.